githubcontent/bips.py

```diff
@@ -0,0 +1,54 @@
+from bs4 import BeautifulSoup
```

Comment:
we need this added to requirements.txt. I've pushed a commit that moves it out of the StackExchange folder to the root.